examples/faster_rcnn/README.md

```diff
@@ -2,11 +2,25 @@
 
 ### Performance
 
-| Training Setting | Evaluation | Reference Implementation | ChainerCV |
+| Training Setting | Evaluation | Reference | ChainerCV |
 |:-:|:-:|:-:|:-:|
 | VOC 2007 trainval | VOC 2007 test|  69.9 mAP [1] | 70.5 mAP |
 
 
+### Speed
+
+Speed of the Caffe implementation and ChainerCV implementation are almost same in the test mode.
```

Comment:
I looked up more on the internet, and it seems that `The speed ... is` is ok,
 too (https://english.stackexchange.com/questions/134703/using-a-singular-or-plural-verb-after-and-or).

I will just say that `ChainerCV and Caffe implementations run at almost the same speed.`